web/src/settings_profile_fields.ts

```diff
@@ -262,24 +281,29 @@ function open_custom_profile_field_form_modal() {
     });
 }
 
-function add_choice_row(e) {
+function add_choice_row(this: HTMLElement, e: JQuery.TriggeredEvent): void {
     const $curr_choice_row = $(this).parent();
     if ($curr_choice_row.next().hasClass("choice-row")) {
         return;
     }
     // Display delete buttons for all existing choices before creating the new row,
     // which will not have the delete button so that there is at least one option present.
     $curr_choice_row.find("button.delete-choice").show();
+    assert(e.delegateTarget instanceof HTMLElement);
     const choices_div = e.delegateTarget;
-    create_choice_row(choices_div);
+    create_choice_row($(choices_div));
```

Comment:
This probably belongs in third commit, but not a big problem ig :)

Reply:
Done. Thanks for reviewing my PR patiently, really appreciate your time in this.